core/src/main/java/feast/core/config/JobConfig.java

```diff
@@ -54,19 +56,34 @@ public JobManager getJobManager(FeastProperties feastProperties)
     String configJson = gson.toJson(runnerConfigOptions);
 
     FeastProperties.MetricsProperties metrics = jobProperties.getMetrics();
+    FeastProperties.StreamProperties streamProperties = feastProperties.getStream();
```

Comment:
Would it be cleaner to move this out into a method?

Reply:
Changed to SpecsStreamingUpdateConfig